docs/DeveloperGuide.md

```diff
@@ -401,16 +423,45 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-**Use case: UC05 - Linking a student to a module class**
+**Use case: UC06 - Linking a student to a class**
+
+**MSS**
+
+1.  User requests to list classes
+2.  Tutor's Pet shows a list of classes
+3.  User requests to list students
+4.  Tutor's Pet shows a list of students
+5.  User requests to link a specific student to a specific class in the list
+6.  Tutor's Pet links the specified student to the specified class
+
+    Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+  Use case ends.
+
+* 4a. The list is empty.
+
+  Use case ends.
+
+* 5a. At least one of the given indexes are invalid.
```

Comment:
Minor grammar issue, should it be `is` and not `are`?
```suggestion
* 5a. At least one of the given indexes is invalid.
```

Reply:
TIL, didn't know that.

Quite interesting explanation here also: [English StackExchange](https://english.stackexchange.com/questions/121531/at-least-one-singular-or-plural-subject/121544)